### PR TITLE
Remove extraneous IVT...

### PR DESCRIPTION
**Pull request by KevinH-MS**

### CODE REVIEW

**Comment by KevinH-MS** [comment]
FYI @jasonmalinowski @tannergooding

File: src/Workspaces/Core/Portable/Workspaces.csproj [hunk 1]
```diff
@@ -250,7 +250,6 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.Features" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.InteractiveEditorFeatures" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.InteractiveFeatures" />
-    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.TestImpact.BuildManager" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.TestImpact.BuildManager.UnitTests" />
```
Comment (jasonmalinowski) [inline src/Workspaces/Core/Portable/Workspaces.csproj]: FYI this should have been InternalsVisbleToTest as well.

**Comment by jasonmalinowski** [comment]
:+1:

File: src/Workspaces/Core/Portable/Workspaces.csproj (hunk 1 above)
Comment (KevinH-MS) [inline src/Workspaces/Core/Portable/Workspaces.csproj]: TestImpact delay-signs tests with the "product" key.

**Pull request merged**